File: src/3423.maximum-difference-between-adjacent-elements-in-a-circular-array.3747/solution.ts
function maxAdjacentDistance (nums: number[]): number {
    const L = nums.length;
    let result = Math.abs(nums[0] - nums[L - 1]);
    for (let i = 1; i < L; i++) {
        result = Math.max(result, Math.abs(nums[i] - nums[i - 1]));
    }
    return result;
}

File: src/3423.maximum-difference-between-adjacent-elements-in-a-circular-array.3747/solution.java
class Solution {
    public int maxAdjacentDistance(int[] nums) {
        int result = Math.abs(nums[0]-nums[nums.length-1]);
        for(int i=1;i< nums.length;i++){
            result = Math.max(result,Math.abs(nums[i]-nums[i-1]));
        }
        return result;
    }
}